pubspec.yaml

```diff
@@ -157,6 +157,7 @@ flutter:
     - assets/images/login/seeds_light_wallet_logo.svg
     - assets/images/receive/receive_input_seeds.svg
     - assets/images/receive/merchant.svg
+    - assets/images/currency_switch_button.png
```

Comment:
why PNG? Can we make an SVG?

Reply:
SVG image for new button is broken, try to fixed in multiple websites, also spoke with Json, he send me new ones
via email but they did not work either, the middle icon seems to be broken. I had to use a png for now.